client/webserver/site/src/js/markets.ts

```diff
@@ -226,7 +226,7 @@ export default class MarketsPage extends BasePage {
     // TODO: Use dexsettings page?
     const registerBttn = Doc.tmplElement(page.notRegistered, 'registerBttn')
     bind(registerBttn, 'click', () => {
-      app().loadPage('register', { host: this.market.dex.host })
+      app().loadPage('register', { host: this.market?.dex?.host })
```

Comment:
Here as well we're now punting an invalid value to another piece of code that will have to deal with undefined.